### PR TITLE
Get rid of dthread

### PR DESCRIPTION
**Pull request by vlolteanu**
I'm 95% certain that dthread is useless. In vanilla, it used to add some delay in between some lower-priority messages on low baud connections. Now it's just a message queue, but all messages are sent asynchronously anyway. So I got rid of it.

A second opinion would be helpful.

### CODE REVIEW

File: Source/multi.cpp [hunk 1]
```diff
@@ -333,14 +332,13 @@ void ProcessTmsgs()
 void SendPlayerInfo(int pnum, _cmd_id cmd)
 {
 	static_assert(alignof(PlayerPack) == 1, "Fix pkplr alignment");
```
Comment (AJenbo) [inline Source/multi.cpp:334]:
```suggestion
	static_assert(alignof(PlayerPack) == 1, "Fix pPack alignment");
```

Also is that still needed?

**Comment by AJenbo** [comment]
Anyone up for helping with play testing this a bit :)

**Review by StephenCWills — DISMISSED** [review]
Nice, I was also wondering why this thread exists. Good riddance.

File: Source/multi.cpp (hunk 1 above)
Comment (vlolteanu) [reply Source/multi.cpp]: No. Fixed.

Comment (StephenCWills) [reply Source/multi.cpp]: Isn't it important for the `reinterpret_cast<byte *>(&pPack)`?

Comment (vlolteanu) [reply Source/multi.cpp]: It would only matter if `alignof(byte)` were larger than `alignof(PlayerPack)`.

**Review by StephenCWills — APPROVED** [review]

**Review by AJenbo — APPROVED** [review]

**Pull request merged**